ext/mbstring/php_mbregex.c

```diff
@@ -474,6 +475,7 @@ static php_mb_regex_t *php_mbregex_compile_pattern(const char *pattern, size_t p
 	if (!rc || onig_get_options(rc) != options || onig_get_encoding(rc) != enc || onig_get_syntax(rc) != syntax) {
 		if ((err_code = onig_new(&retval, (OnigUChar *)pattern, (OnigUChar *)(pattern + patlen), options, enc, syntax, &err_info)) != ONIG_NORMAL) {
 			onig_error_code_to_str(err_str, err_code, &err_info);
+			// Convert to error?
```

Comment:
Probably not. Checking whether a regex is valid by trying to compile it and inspecting the return value is a legitimate use-case (at least I've seen this done with preg_*).

Reply:
ACK